src/plugins/bodhi.c

```diff
@@ -513,9 +510,9 @@ int main(int argc, char **argv)
     if (!bugs && !argv[optind])
         libreport_show_usage_and_die(program_usage_string, program_options);
 
-    struct strbuf *query = libreport_strbuf_new();
+    GString *query = g_string_new(NULL);
```

Comment:
You can also get automatic cleanup for `GString` (https://gitlab.gnome.org/GNOME/glib/-/blob/master/glib/glib-autocleanups.h).